android/src/test/java/com/instabug/flutter/InstabugApiTest.java

```diff
@@ -345,6 +347,32 @@ public void testClearAllExperiments() {
         mInstabug.verify(Instabug::clearAllExperiments);
     }
 
+    @Test
+    public void testAddFeatureFlags() {
+       Map<String,String > featureFlags = new HashMap<>();
+        featureFlags.put("key1","variant1");
+        api.addFeatureFlags(featureFlags);
+        List<IBGFeatureFlag> s=new ArrayList<IBGFeatureFlag>();
```

Comment:
It is recommended to use descriptive variable names instead of using single letters like "s" in this case.